base/src/main/java/org/wildfly/security/WildFlyElytronBaseProvider.java

```diff
@@ -26,7 +26,6 @@
 import java.lang.reflect.Constructor;
 import java.security.NoSuchAlgorithmException;
 import java.security.Provider;
-import java.security.Provider.Service;
```

Comment:
I just wonder how the checkstyle was passing previously if this is unused, but seems ok